test/NuGet.Core.Tests/NuGet.Commands.Test/RestoreCommandTests/RestoreCommandTests.cs

```diff
@@ -1465,7 +1466,7 @@ public async Task RestoreCommand_MinimalProjectWithAdditionalMessages_WritesAsse
         }
 
         [Fact]
-        public async Task RestoreCommand_CentralVersion_ErrorWhenDependenciesHaveVersion()
+        public async Task RestoreCommand_CentralPackageManagement_ErrorWhenDependenciesHaveVersion()
```

Comment:
nit: If this were a new test I'd definitely suggest naming without the class name repeated in the method name.
You can decide if it should apply to the refactoring.
Overall, I had to read the test a few times to understand what this test title meant.
eg,
```suggestion
        public async Task ExecuteAsync_CentralPackageManagementEnabled_WhenPackageReferencesHaveVersion_LogsAnError()
```